examples/device/ep/tests/utils.py

```
@@ -221,17 +225,23 @@ def bench_kineto(
-        prof.key_averages()
-        .table(sort_by="cuda_time_total", max_name_column_width=100)
-        .split("\n")
+    prof_table = prof.key_averages().table(
+        sort_by="cuda_time_total", max_name_column_width=100
     )
+
+    if "CUDA" not in prof_table:
+        raise KinetoUnavailableError(
+            "Kineto recorded no CUDA timing activities; "
+            "per-kernel profiling is unavailable"
+        )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)